packages/framework/src/Component/Cron/MutexFactory.php

```diff
@@ -33,10 +33,28 @@ public function __construct(LockInterface $lock)
      */
     public function getCronMutex()
```

Comment:
should not we mark this method as deprecated as well? And possibly remove it in the closest major release?

Reply:
Can be. At first, I thought about this class as an independent "service", but as long as it is in `Cron` namespace and it's not used anywhere else, it should be probably deprecated.